src/node/server/serverPluginModuleRewrite.ts

```diff
@@ -160,6 +160,10 @@ export function rewriteImports(
         let id = source.substring(start, end)
         let hasLiteralDynamicId = false
         if (dynamicIndex >= 0) {
+          // #998 remove comment
+          id = id.replace(/\/\*[\s\S]*?\*\/|([^\\:]|^)\/\/.*$/gm, '')
+          // remove spaces
+          id = id.replace(/\s/g, '')
```

Comment:
I think this line can be removed by changing the match regex below to account for potential spaces